apps/src/aichat/types.ts

```diff
@@ -5,41 +5,44 @@ import {Role} from '../aiComponentLibrary/chatItems/types';
 // TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
 export type AichatInteractionStatusValue = string;
 
-export interface ChatItem {
+export interface AichatEvent {
   // UTC timestamp in milliseconds
   timestamp: number;
+  description?: string;
```

Comment:
What is `description` used for / planned to be used for?

Reply:
This will be the text to display in the teacher view of student chat history for events that are not chat messages. Example: 'User cleared chat history.'

Reply:
I can add a comment including this.

Reply:
That makes sense. I like the idea of discrete event types since I agree there won't be many of them.

Reply:
To make the view of the `AichatEvent` more simple,  I opted for storing a string key instead. Lmk what you think.